### DISCUSSION TITLE
Android app empty

### DISCUSSION THREAD

**Discussion by SerdaOzun** [immich-app/immich#169] [Q&A] [answered]
Hey,
I wanted to try out the project as it's the most promising one yet! I've set the server up correctly and can access it. Downloaded the latest version of the android app and also gave it permissions, but it just remains empty and is not able to find any pictures

[Screenshot_2022-05-22-00-25-22-738_app alextran immich](https://user-images.githubusercontent.com/46612292/169671002-ccec7b28-6a78-4590-8db0-0ec04f85b0a7.jpg)

Am I missing something? thanks

Xiaomi mi 9 lite running Android 10

Edit: Oh well, solve this quickly. I didn't realize that only backed up pictures are shown there :) Also apparently I hadn't rechecked clicking on 'select' next to backup albums after giving permissions to the app. Works great now :grin: 
Since you are working on recreating a google photos clone, will you be adding the option to also show local photos there? And also just using the local files if present. It seems that the server is being accessed for viewing images which already are on the phone

**Comment by alextran1502** [comment, marked as answer]
Hello, I thought about this and did some experimentation. But this application goal is a little bit different from Google Photos, my primary goal is to make it a backup tool, the additional photo viewer, and photo gallery features come after the first goal. So to make it as performance and as simple as possible, I've decided not to show local photos on the main dashboard :)
